### PR TITLE
Tutorial modularization

### PR DESCRIPTION
**Pull request by trav-d13**
The tutorial was modularized which should enable us to add skills and functionality to the assistant. 

Additionally all method are documented. 

Please build and test your side before approving to check the docs and the assistant is working for you

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,13 +32,21 @@ goes to their official site: https://docs.conda.io/en/latest/
...
 
-### Documentation
+## Documentation
 [Sphinx Documents](https://www.sphinx-doc.org/en/master/) allows for comprehensive documentation of the methods and
  packages within the K_voice_assist project. 
 
-#TODO Create documentation and information on how to create/ use it
+#### Generating Docs
+1. In terminal access the docs directory `cd docs`
+2. If you have added any additional files in any directory, run the respective command below:
```
Comment (ericbanzuzi) [inline README.md:46]: should this (2.) still be in here? you removed it in the previous PR

File: k_assist.yml [hunk 2]
```diff
@@ -3,4 +3,12 @@ channels:
   - defaults
 dependencies:
   - python=3.10
+  - speechrecognition
```
Comment (ericbanzuzi) [inline k_assist.yml:6]: can you move these (after python) inside pip so it works for me as well, like this:
- pip:
  - speechrecognition
  - wikipedia
  - pyjokes
  - sphinx
  - sphinxcontrib-napoleon
  - sphinx_rtd_theme
  - sphinx-autodoc-typehints
  - pyaudio

**Review by ericbanzuzi — CHANGES_REQUESTED** [review]
looks good and is working for me except the k_assist.yml, I left a few comments how this could be fixed (it worked for me).

File: README.md (hunk 1 above)
Comment (trav-d13) [reply README.md:46]: Yeah, I thought it wasn't in the prev PR, but it adds the new files into the docs

File: k_assist.yml (hunk 2 above)
Comment (trav-d13) [reply k_assist.yml:6]: I just made the changes, but am getting errors my side. How are you trying to create the environment?

Comment (trav-d13) [reply k_assist.yml]: Fixed

**Review by ericbanzuzi — APPROVED** [review]
Everything works fine now!

**Pull request merged**